plugins/dependencies/src/main/resources/removed-dependencies-versions-keys.txt

```diff
@@ -51,3 +51,36 @@ androidx.legacy..legacy-support-v4=version.androidx.legacy
 androidx.hilt..hilt-lifecycle-viewmodel=version.androidx.hilt-lifecycle-viewmodel
 androidx.car..car-cluster=version.androidx.car-cluster
 androidx.car..car=version.androidx.car
+io.reactivex.rxjava2..rxjava2=version.NO-RULE
+io.reactivex.rxjava3..rxjava3=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-appcompat=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-core=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-drawerlayout=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-leanback=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-material=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-recyclerview=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-slidingpanelayout=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-swiperefreshlayout=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-viewpager2=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding-viewpager=version.NO-RULE
+com.jakewharton.rxbinding3..rxbinding=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-appcompat=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-core=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-drawerlayout=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-leanback=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-material=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-recyclerview=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-slidingpanelayout=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-swiperefreshlayout=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-viewpager2=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding-viewpager=version.NO-RULE
+com.jakewharton.rxbinding4..rxbinding=version.NO-RULE
+com.jakewharton.rxrelay2..rxrelay=version.NO-RULE
+com.jakewharton.rxrelay3..rxrelay=version.NO-RULE
+io.reactivex..rxjava=version.NO-RULE
+io.reactivex.rxjava2..rxandroid=version.NO-RULE
+io.reactivex.rxjava2..rxjava=version.NO-RULE
+io.reactivex.rxjava2..rxkotlin=version.NO-RULE
+io.reactivex.rxjava3..rxandroid=version.NO-RULE
+io.reactivex.rxjava3..rxjava=version.NO-RULE
+io.reactivex.rxjava3..rxkotlin=version.NO-RULE
```

Comment:
You can remove all these lines, and re-run the tests, you'll see they are not added back. This was caused by a tests run order fact that wasn't taken into account, which is now fixed on the main branch.